x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotRecoveryStateIntegrationTests.java

```diff
@@ -148,4 +237,29 @@ private void assertExecutorIsIdle(String executorName) throws Exception {
     private DiscoveryNodes getDiscoveryNodes() {
         return client().admin().cluster().prepareState().clear().setNodes(true).get().getState().nodes();
     }
+
+    /**
+     * A fs repository plugin that allows using its methods from any thread
+     */
+    public static class TestRepositoryPlugin extends Plugin implements RepositoryPlugin {
+        @Override
+        public Map<String, Repository.Factory> getRepositories(
+            Environment env,
+            NamedXContentRegistry namedXContentRegistry,
+            ClusterService clusterService,
+            BigArrays bigArrays,
+            RecoverySettings recoverySettings
+        ) {
+            return Collections.singletonMap(
+                "test-fs",
+                (metadata) -> new FsRepository(metadata, env, namedXContentRegistry, clusterService, bigArrays, recoverySettings) {
+                    @Override
+                    protected void assertSnapshotOrGenericThread() {
+                        // ignore
+                    }
+
```

Comment:
nit: extra line

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/67425/commits/e0c5c8d9eb3b271d669bdedc239ca255b0d510b6